### PR TITLE
adjustments to fix CI jobs

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by dhellmann** [comment]
This should move us closer to having working CI jobs.

@sadasu could you download the PR and run the tests locally to make sure I have everything in the commit that we need, please?

/cc @deads2k we will need another admin-merge for this one until we have prow fully configured

File: Makefile [hunk 1]
```diff
@@ -44,7 +47,11 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ./...
+	hack/go-fmt.sh
+
+# Run go lint against code
+lint:
+	golint -set_exit_status -min_confidence 0.3 ./...
```
Comment (sadasu) [inline Makefile:54]: When I ran "make lint" after I performed "go mod vendor", I found that go-lint was running on all files in the /vendor directory too. 
Tried the following locally, and it seemed to work.

      golint -set_exit_status -min_confidence 0.3 $(go list -f '{{ .ImportPath }}' ./...)

Comment (dhellmann) [reply Makefile]: Good catch. Fixed.

File: go.mod [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
-	k8s.io/apimachinery v0.0.0-20190913080033-27d36303b655
-	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
-	sigs.k8s.io/controller-runtime v0.4.0
+	github.com/onsi/ginkgo v1.12.0
+	github.com/onsi/gomega v1.8.1
+	github.com/openshift/api v0.0.0-20200424083944-0422dc17083e
+	k8s.io/api v0.18.2 // indirect
```
Comment (sadasu) [inline go.mod:10]: When I ran go mod vendor; go mod tidy; go mod verify,  k8s.io/api was removed from my go.mod file.

Comment (dhellmann) [reply go.mod]: Fixed

**Comment by deads2k** [comment]
> /lgtm

manually merging to help get CI going.

**Pull request merged**